packages/wonder-stuff-i18n/tsconfig.json

```diff
@@ -4,5 +4,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "rootDir": "src",
-  }
+  },
+  "references": [
+  ]
```

Comment:
**note:** Remnants from trying to fix a small nit. Seemed better to leave it for now as it makes adding references easier in the long term and its inert.